package.json

```diff
@@ -6,11 +6,12 @@
   ],
   "scripts": {
     "prebuild": "npm run build --workspace=@amsterdam/asc-assets",
-    "build": "npm run clean:generated && npm run build:ts && npm run build:ts:es",
-    "build:watch": "npm run clean:generated && concurrently \"npm run build:ts -- --watch\" \"npm run build:ts:es -- --watch\"",
+    "build": "npm run clean:generated && npm run build:ts && npm run build:ts:es && npm run build:ts:beta",
+    "build:watch": "npm run clean:generated && concurrently \"npm run build:ts -- --watch\" \"npm run build:ts:es -- --watch\" \"npm run build:ts:beta -- --watch\"",
```

Comment:
Wouldn't it be better to split this into two separate scripts? Might be a lot to watch for if the `beta` folder grows and, when developing, I don't think you even need to watch for changes in both.